libs/openFrameworks/utils/ofUtils.cpp

```diff
@@ -469,10 +454,7 @@ bool ofToBool(const string& boolString) {
 	if(lower == "false") {
 		return false;
 	}
-	bool x = false;
-	istringstream cur(lower);
-	cur >> x;
-	return x;
+	return (bool)std::stoi(boolString);
```

Comment:
yeah, you're right!!
i had been a misunderstanding.

but, we need a try-catch around `std::stoi` for case of `lower` is neither "true" and "false".
i fix again.